automation/jenkins_build.sh

```diff
@@ -58,11 +58,11 @@ mkdir -p $BUILD_DEPLOY_DIR
 rm -rf $BUILD_DEPLOY_DIR/* # do we have anything there?
 mv -v $(readlink --canonicalize $WORKSPACE/build/tmp/deploy/images/$MACHINE/$DEPLOY_ARTIFACT) $BUILD_DEPLOY_DIR/$DEPLOY_ARTIFACT
 if [ "${COMPRESSED}" == 'true' ]; then
-	if [ "${ARCHIVE}" == 'true' ]; then
-		(cd $BUILD_DEPLOY_DIR && tar --remove-files -zcvf ${DEPLOY_ARTIFACT}.tar.gz $DEPLOY_ARTIFACT)
-	else
-		gzip $BUILD_DEPLOY_DIR/$DEPLOY_ARTIFACT
+	if [ "${ARCHIVE}" != 'true' ]; then
```

Comment:
This is whether yocto generates an archive or not, ie a directory - if it's not a directory then we need to standardise the name to resin.img to play nicely with automatically extracting with the right filename (since it has to be a tar file to be automatically decompressed)